tests/zfs-tests/tests/functional/zvol/zvol_misc/zvol_misc_snapdev.ksh

```diff
@@ -148,6 +148,12 @@ log_must zfs set snapdev=hidden $ZVOL
 check_missing $SNAPDEV
 log_must zfs set snapdev=hidden $ZVOL
 check_missing $SNAPDEV
+# 2.4 Verify we don't hit a race condition setting snapdev in rapid succession
```

Comment:
Unfortunately only `udevadm settle` doesn't seem to fix this completely: calling `block_device_wait` would "fix" the test case but render it useless, since also the original zv_open_count race cannot be triggered with `udevadm trigger` + `udevadm settle`.